api/geth_backend.go

```
@@ -1751,7 +1751,12 @@ func (b *GethStatusBackend) loadNodeConfig(inputNodeCfg *params.NodeConfig) erro
 	conf.Version = params.Version
 	conf.RootDataDir = b.rootDataDir
 	conf.DataDir = filepath.Join(b.rootDataDir, conf.DataDir)
-	conf.ShhextConfig.BackupDisabledDataDir = filepath.Join(b.rootDataDir, conf.ShhextConfig.BackupDisabledDataDir)
+	if !filepath.IsAbs(conf.ShhextConfig.BackupDisabledDataDir) {
+		conf.ShhextConfig.BackupDisabledDataDir = filepath.Join(b.rootDataDir, conf.ShhextConfig.BackupDisabledDataDir)
+	}
+	if !filepath.IsAbs(conf.ShhextConfig.BackupDisabledDataDir) {
+		b.log.Error("BackupDisabledDataDir was expected to be absolute path, instead it is ", conf.ShhextConfig.BackupDisabledDataDir)
```

Do we need to return an error here instead of letting the flow continue?